frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -265,21 +291,26 @@ pods:
         cmd: |
           set -e
           export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/)
-          cp -v hdfs {{TASKCFG_ALL_HDFS_VERSION}}/bin/
```

Comment:
why'd you delete this and every where else? No longer exists in hdfs vs cdh?

Reply:
Latest version of both Apache and CDH also contains `hdfs` script. But, it has completely changed compared to 2.6.0. In the previous version additional logic was applied:
`bin=`dirname $0``
`bin="$MESOS_SANDBOX/${bin}"` in the hdfs script. That's why it was copied here. In the case of latest version, everything works well without applying this logic.